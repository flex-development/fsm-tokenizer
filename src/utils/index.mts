/**
 * @file Entry Point - Utilities
 * @module fsm-tokenizer/utils
 */

export { default as eof, default as eos } from '#utils/eof'
export { default as eol } from '#utils/eol'
export { default as isCode } from '#utils/is-code'
export { default as push } from '#utils/push'
export { default as resolveAll } from '#utils/resolve-all'
export { default as serializeChunks } from '#utils/serialize-chunks'
export { default as sliceChunks } from '#utils/slice-chunks'
export { default as sliceSerialize } from '#utils/slice-serialize'
export { default as splice } from '#utils/splice'
export { default as htab, default as tab } from '#utils/tab'
