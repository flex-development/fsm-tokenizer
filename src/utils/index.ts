/**
 * @file Entry Point - Utilities
 * @module vfile-lexer/utils
 */

export { default as isLineEnding } from './is-line-ending'
export { default as resolveAll } from './resolve-all'
export { default as resolveSlice } from './resolve-slice'
