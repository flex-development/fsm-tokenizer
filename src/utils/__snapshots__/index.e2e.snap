// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`e2e:utils > should expose public api 1`] = `
[
  "eof",
  "eos",
  "eol",
  "isCode",
  "push",
  "resolveAll",
  "serializeChunks",
  "sliceChunks",
  "sliceSerialize",
  "splice",
  "htab",
  "tab",
]
`;
